README.md

```diff
@@ -29,6 +29,18 @@ See the [installation guide](https://book.getfoundry.sh/getting-started/installa
 
 If you're experiencing any issues while installing, check out [Getting Help](#getting-help) and the [FAQ](https://book.getfoundry.sh/faq).
 
+As a quick guide, if you would like to install locally you can use the following bash command:
+
+```sh
+for pkg in anvil forge; do cargo install --path ./crates/$pkg --profile local --force; done
+```
+
+or if you have fish
+
+```fish
```

Comment:
I apologize for sending a useless for loop to you :disappointed: for copy paste instructions it would be fine to just have two lines like we did before

```shell
cargo install --path ./crates/forge --profile local --force
cargo install --path ./crates/anvil --profile local --force
```

But you can merge as is if you want. I'll probably rewrite this README soon anyways.

Reply:
Oh I kinda liked the loop :). But I'll switch it to the two commands instead.